Twispay WooCommerce Plugin/twispay/helpers/Twispay_TW_Status_Updater.php

```diff
@@ -141,7 +141,7 @@ public static function updateStatus_backUrl($orderId, $serverStatus, $checkout_u
                       $subscription = reset($subscription);
 
                       /* First payment on order, process payment & activate subscription. */
-                      if ( 0 == $subscription->get_completed_payment_count() ) {
+                      if ( 0 == $subscription->get_payment_count() ) {
```

Comment:
When I received IPN (server to server notification) I've got `<b>Fatal error</b>:  Uncaught Error: Call to a member function get_payment_count() on bool in 
 wordpress/wp-content/plugins/twispay/helpers/Twispay_TW_Status_Updater.php:261`

the $subscription (on my test) looks like:
`array(1) {
  [23]=>
  bool(false)
}`

so, `reset($subscription)` return false (bool)

Reply:
This one isn't actually related to my changes, I only changed the deprecated function to a recommended one. So, what do we actually want to do in case if `$subscription` is empty?

Reply:
@carco go through my comments and you will have the hints to solve your issue https://github.com/Twispay/WooCommerce/issues/17